perception/autoware_radar_object_clustering/src/radar_object_clustering_node.hpp

```diff
@@ -52,7 +52,7 @@ class RadarObjectClusteringNode : public rclcpp::Node
   rclcpp::Subscription<DetectedObjects>::SharedPtr sub_objects_{};
 
   // Callback
-  void onObjects(const DetectedObjects::ConstSharedPtr msg);
+  void onObjects(const DetectedObjects::ConstSharedPtr objects_data_);
```

Comment:
I think it's better to keep the original declaration and rename the .cpp function instead.

Reply:
Thanks for the advice.
I removed objects_data_ and unified it with msg.

This response is the same as [PR#8013](https://github.com/autowarefoundation/autoware.universe/pull/8013).